### PR TITLE
Fix more warnings

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: MediaBrowser.sln [hunk 1]
```diff
@@ -41,8 +41,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Emby.Naming", "Emby.Naming\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Emby.XmlTv", "Emby.XmlTv\Emby.XmlTv\Emby.XmlTv.csproj", "{6EAFA7F0-8A82-49E6-B2FA-086C5CAEA95B}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IsoMounter", "Emby.IsoMounting\IsoMounter\IsoMounter.csproj", "{9BA471D2-6DB9-4DBF-B3A0-9FB3171F94A6}"
```
Comment (EraYaN) [inline MediaBrowser.sln]: If this a artifact from the PR that moved this to a plugin? Is this supposed to be in this PR?

Comment (Bond-009) [reply MediaBrowser.sln]: Apparently this wasn’t changed in that PR, so current master is failing. This could be a separate PR. 🤷‍♂️

Comment (EraYaN) [reply MediaBrowser.sln]: Mmm might as well have it in this one. It's a "warning" right?

Comment (Bond-009) [reply MediaBrowser.sln]: It’s a build failure, when trying to build the sln, building the startup project works fine.

**Comment by anthonylavado** [comment]
Verified CI issue is just because some methods are modified.

**Comment by anthonylavado** [comment]
There will be a minor conflict with some of the changes in #1643 ,where Entrypoint.cs and AlbumNFOSaver.cs are both edited, with some of the same fixes seen in this PR, but slightly differently.

**Pull request merged**